doc/application/index.rst

```diff
@@ -1123,15 +1120,20 @@ Make sure to follow these steps in order.
    found elsewhere if the CMake variable :makevar:`KCONFIG_ROOT` is
    set with an absolute path.
 
-#. Now include the mandatory boilerplate that integrates the
-   application with the Zephyr build system on a new line, **after any
+#. Specify that the application requires Zephyr on a new line, **after any
    lines added from the steps above**:
 
    .. code-block:: cmake
 
-      include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+      find_package(Zephyr)
```

Comment:
Add a small section about backwards-compatibility with the old` include(... boilerplate)` line

Reply:
done
```
.. note::

include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE) is
still supported for backward compatibility with older applications.
Including boilerplate.cmake directly in the sample still requires to run
source zephyr-env.sh or execute zephyr-env.cmd before building the application.
```